### PR TITLE
Download page

### PR DESCRIPTION
**Pull request by mitjapotocin**
@markotoplak download page will have the content inside html, if you are ok with it.

### CODE REVIEW

File: pages/download/index.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          <p>
+            Orange can also be installed from the Python Package Index. You may need additional system packages provided
+            by your distribution.
+          </p>
+          <code>pip install orange3</code>
+
+          <h3>Installing from source</h3>
+          <p>
+            Clone our repository from GitHub or download the source code tarball. Then follow the instructions in
+            README.md
```
Comment (mitjapotocin) [inline pages/download/index.tsx:63]: @markotoplak i guess there is a typo in the design `source code tarball`.
This should be a link to `https://github.com/biolab/orange3`?

File: pages/download/index.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          <h3>Pip</h3>
+          <p>
+            Orange can also be installed from the Python Package Index. You may need additional system packages provided
+            by your distribution.
+          </p>
+          <code>pip install orange3</code>
+
+          <h3>Installing from source</h3>
+          <p>
+            Clone our repository from GitHub or download the source code tarball. Then follow the instructions in
```
Comment (mitjapotocin) [inline pages/download/index.tsx:62]: @markotoplak is `tarball` a typo?

**Comment by mitjapotocin** [comment]
@teaskvarc /download page is ready.

File: pages/download/index.tsx (hunk 2 above)
Comment (markotoplak) [reply pages/download/index.tsx]: That used to be a common term. :)

**Comment by markotoplak** [comment]
Thanks!

**Pull request merged**